src/RTCDTMFSender.ts

```diff
@@ -0,0 +1,55 @@
+import { NativeModules } from 'react-native';
+
+const { WebRTCModule } = NativeModules;
+const DEFAULT_DURATION = 300;
+const DEFAULT_INTER_TONE_GAP = 70;
+const MIN_DURATION = 70;
+const MAX_DURATION = 6000;
+const MIN_INTER_TONE_GAP = 50;
+
+export default class RTCDTMFSender {
```

Comment:
Even though we don't implement the `tonechange` event, we should extend from event target here.